modules/Dash/resources/dash.js

```diff
@@ -521,6 +521,7 @@ try {
 				};
 				if (error.category) {
 					errorObj.code = error.category + "000";
+					errorObj.key = error.category + "000";
```

Comment:
```suggestion
					errorObj.key = errorObj.code";
```
but see my Q in kAnalony.js

Reply:
yes, same end result and there's no logic here.